:  failed

. [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19921555?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_LintURLsBazel/19921555?buildTab=artifacts#/) on master @ [be2cc319eac7854920c9a4f5f5cb8a1e3a643e71](https://github.com/cockroachdb/cockroach/commits/be2cc319eac7854920c9a4f5f5cb8a1e3a643e71):


```
[2,214 / 2,622] GoLink external/co_honnef_go_tools/cmd/staticcheck/staticcheck_/staticcheck; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,241 / 2,622] GoCompilePkg external/com_github_cockroachdb_pebble/sstable/sstable.a; 1s processwrapper-sandbox ... (4 actions, 3 running)
[2,261 / 2,622] GoCompilePkg pkg/util/log/log.a; 1s processwrapper-sandbox ... (8 actions, 7 running)
[2,274 / 2,622] GoCompilePkg pkg/util/log/log.a; 2s processwrapper-sandbox ... (8 actions, 7 running)
[2,294 / 2,622] Running nogo on //pkg/sql/sessiondatapb:sessiondatapb; 2s processwrapper-sandbox ... (8 actions, 7 running)
[2,329 / 2,622] GoCompilePkg pkg/roachpb/roachpb.a; 1s processwrapper-sandbox ... (5 actions running)
[2,332 / 2,622] GoCompilePkg pkg/roachpb/roachpb.a; 2s processwrapper-sandbox ... (5 actions running)
[2,338 / 2,622] GoCompilePkg pkg/sql/sem/tree/tree.a; 3s processwrapper-sandbox ... (8 actions running)
[2,368 / 2,622] GoCompilePkg pkg/sql/sem/tree/tree.a; 5s processwrapper-sandbox ... (3 actions running)
[2,387 / 2,622] GoCompilePkg pkg/kv/kvpb/kvpb.a; 0s processwrapper-sandbox ... (8 actions, 7 running)
[2,398 / 2,622] GoCompilePkg pkg/kv/kvpb/kvpb.a; 2s processwrapper-sandbox ... (4 actions running)
[2,399 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 5s processwrapper-sandbox ... (6 actions, 3 running)
[2,406 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 6s processwrapper-sandbox ... (8 actions, 7 running)
[2,415 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 7s processwrapper-sandbox ... (8 actions, 7 running)
[2,423 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 8s processwrapper-sandbox ... (8 actions running)
[2,430 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 9s processwrapper-sandbox ... (7 actions, 4 running)
[2,433 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 11s processwrapper-sandbox ... (4 actions running)
[2,434 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 16s processwrapper-sandbox ... (3 actions running)
[2,436 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 17s processwrapper-sandbox ... (3 actions running)
[2,441 / 2,622] Running nogo on //pkg/kv/kvpb:kvpb; 18s processwrapper-sandbox ... (6 actions, 5 running)
[2,446 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 19s processwrapper-sandbox ... (4 actions, 3 running)
[2,462 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 20s processwrapper-sandbox ... (8 actions, 7 running)
[2,479 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 21s processwrapper-sandbox ... (8 actions, 7 running)
[2,511 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 22s processwrapper-sandbox ... (5 actions running)
[2,522 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 23s processwrapper-sandbox ... (4 actions running)
[2,525 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 25s processwrapper-sandbox ... (3 actions, 2 running)
[2,527 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 26s processwrapper-sandbox ... (3 actions, 2 running)
[2,527 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 27s processwrapper-sandbox ... (3 actions running)
[2,530 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 29s processwrapper-sandbox ... (2 actions, 1 running)
[2,530 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 30s processwrapper-sandbox ... (2 actions running)
[2,534 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 32s processwrapper-sandbox ... (5 actions, 4 running)
[2,546 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 33s processwrapper-sandbox ... (3 actions running)
[2,548 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 35s processwrapper-sandbox ... (3 actions, 2 running)
[2,556 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 36s processwrapper-sandbox ... (2 actions running)
[2,566 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 37s processwrapper-sandbox ... (3 actions running)
[2,574 / 2,622] GoCompilePkg pkg/sql/parser/parser.a; 39s processwrapper-sandbox
[2,575 / 2,622] [Prepa] GoCompilePkg pkg/util/jsonpath/parser/parser.a
[2,584 / 2,622] [Prepa] GoCompilePkg pkg/sql/sem/normalize/normalize.a ... (7 actions, 0 running)
[2,607 / 2,622] GoCompilePkg pkg/sql/sem/asof/asof.a; 0s processwrapper-sandbox ... (3 actions, 1 running)
[2,611 / 2,622] Running nogo on //pkg/sql/sem/builtins:builtins; 0s processwrapper-sandbox ... (2 actions running)
[2,613 / 2,622] GoCompilePkg pkg/sql/sem/builtins/builtins.a; 1s processwrapper-sandbox
[2,614 / 2,622] [Prepa] Running nogo on //pkg/testutils/lint:lint_test
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/testutils/lint/BUILD.bazel:15:8: GoCompilePkg pkg/testutils/lint/lint_test.internal.a failed: (Exit 1): builder failed: error executing GoCompilePkg command (from target //pkg/testutils/lint:lint_test) bazel-out/k8-opt-exec-ST-a828a81199fe/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -goroot bazel-out/k8-fastbuild/bin/external/io_bazel_rules_go/stdlib_ -installsuffix ... (remaining 61 arguments skipped)

Use --sandbox_debug to see verbose messages from the sandbox and retain the sandbox build root for debugging
pkg/testutils/lint/nightly_lint_test.go:28:68: build.ParsedVersion().IsCustomOrNightlyBuild undefined (type *"github.com/cockroachdb/version".Version has no field or method IsCustomOrNightlyBuild)
compilepkg: error running subcommand external/go_sdk/pkg/tool/linux_amd64/compile: exit status 2
Target //pkg/testutils/lint:lint_test failed to build
Use --verbose_failures to see the command lines of failed build steps.
ERROR: /go/src/github.com/cockroachdb/cockroach/pkg/testutils/lint/BUILD.bazel:15:8 Validating nogo output for //pkg/testutils/lint:lint_test failed: (Exit 1): builder failed: error executing GoCompilePkg command (from target //pkg/testutils/lint:lint_test) bazel-out/k8-opt-exec-ST-a828a81199fe/bin/external/go_sdk/builder_reset/builder compilepkg -sdk external/go_sdk -goroot bazel-
```
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/test-eng
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-51491

--- a/pkg/testutils/lint/nightly_lint_test.go
+++ b/pkg/testutils/lint/nightly_lint_test.go
@@ -25,7 +25,7 @@ func TestNightlyLint(t *testing.T) {
 	// TestHelpURLs checks that all help texts have a valid documentation URL.
 	t.Run("TestHelpURLs", func(t *testing.T) {
 		skip.UnderShort(t)
-		if build.ParsedVersion().IsPrerelease() || build.ParsedVersion().IsCustomOrNightlyBuild() {
+		if build.ParsedVersion().IsPrerelease() || build.ParsedVersion().IsCustomOrAdhocBuild() {
 			skip.IgnoreLint(t, "pre-release or customized build")
 		}
 		if pkgSpecified {

